src/model/queries/get_data.js

```diff
@@ -0,0 +1,16 @@
+const dbConnection = require('../database/db_connection');
+
+const checkUser = (email) => new Promise((resolve, reject) => {
+    const sql = {
+        text: "SELECT * FROM users WHERE email=$1;",
+        values: [email]
+    };
+    dbConnection.query(sql, (error, res) => {
+        if (error) {
+          return reject(new TypeError("Error in DB"));
+        }
+        resolve(res.rows);
```

Comment:
consistent return. If you use return inside a function you should keep using it.

```js
return resolve(res.rows)
```